acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/cluster/Cluster.java

```diff
@@ -87,12 +87,19 @@ public void start(final List<? extends RunnableNode> nodes) {
 
     nodes
         .parallelStream()
-        .filter(node -> bootnode.map(boot -> boot != node).orElse(true))
+        .filter(
```

Comment:
doing this in a custom fork-join pool can be done like:
`
ForkJoinPool forkJoinPool = new ForkJoinPool(nodes.size() - 1);
forkJoiNPool.submit(nodes.parallelStream() ....
`
That removes a dependency on the "system wide" thread pool, which removes a dependency which might be causing problems,

Reply:
would it make sense to do this as a separate PR in case it causes problems? Cluster is used by lots of tests

Reply:
sure - we can do that.

Reply:
there are some tests that start cluster with one node - and sometimes there's more than one bootnode. so it's not that neat.